### PR TITLE
Misc Promethus changes

### PR DESCRIPTION
**Pull request by tillprochaska**
This PR collects a few minor, miscellaneous changes related to Prometheus metrics, based on my (limited) practical experience integrating Prometheus into other Aleph components.

### CODE REVIEW

File: servicelayer/worker.py [hunk 1]
```diff
@@ -22,26 +29,29 @@
 INTERVAL = 2
 TASK_FETCH_RETRY = 60 / INTERVAL
 
-TASK_STARTED = Counter(
-    "task_started_total",
+REGISTRY.unregister(GC_COLLECTOR)
+REGISTRY.unregister(PROCESS_COLLECTOR)
```
Comment (stchris) [inline servicelayer/worker.py:33]: Are these not useful to us?

File: servicelayer/worker.py [hunk 2]
```diff
@@ -143,14 +153,14 @@ def retry(self, task):
...
-            TASK_FAILED.labels(task.stage.stage, retries, False).inc()
+            TASKS_FAILED.labels(task.stage.stage, retries, False).inc()
             task.context["retries"] = retry_count
             task.stage.queue(task.payload, task.context)
         else:
             log.warning(
                 f"Failed task, exhausted retry count of {settings.WORKER_RETRY}"
             )
-            TASK_FAILED.labels(task.stage.stage, retries, True).inc()
+            TASKS_FAILED.labels(task.stage.stage, retries, True).inc()
```
Comment (stchris) [inline servicelayer/worker.py:163]: For the sake of maintainabilty looking forward, could we use keyword arguments to make this a bit more readable? ([The docs](https://prometheus.github.io/client_python/instrumenting/labels/) suggest this works)

```suggestion
            TASKS_FAILED.labels(stage=task.stage.stage, retries=retries, failed_permanently=True).inc()
```

**Review by stchris — COMMENTED** [review]
Generally agree with this, I just have two questions

File: servicelayer/worker.py (hunk 2 above)
Comment (tillprochaska) [reply servicelayer/worker.py:163]: Oh yes, absolutely. I’ve been wondering about that before, not sure how I’ve missed that… 🙃 I’ll make the change in servicelayer and also in ingest-file and Aleph.

File: servicelayer/worker.py (hunk 1 above)
Comment (tillprochaska) [reply servicelayer/worker.py:33]: I’ve removed the process collector as the metrics it provides are probably a bit redundant with regards to the metrics you usually already collect in a containerized environment (CPU time, memory usage, …)

With regards to garbage collection, I’m honestly not sure -- this is usually something I don't mess with in Python.

Both collectors don’t work well in multi-process mode (as they are per-process metrics), so they are not available for the API service, but I’m happy to add them back for the Aleph and ingest-file workers.

File: servicelayer/worker.py (hunk 2 above)
Comment (tillprochaska) [reply servicelayer/worker.py]: @stchris Done! :)

Comment (stchris) [reply servicelayer/worker.py]: Thanks, Till!

File: servicelayer/worker.py (hunk 1 above)
Comment (stchris) [reply servicelayer/worker.py:33]: Thanks for explaining this!

**Review by stchris — APPROVED** [review]

**Pull request merged**